easybuild/easyconfigs/t/Trim_Galore/Trim_Galore-0.6.2-GCCcore-8.2.0.eb

```diff
@@ -0,0 +1,46 @@
+# Contribution from the Crick HPC team
+# uploaded by J. Sassmannshausen
+#
+# Updated: Pavel Grochal (INUITS)
+#
+
+easyblock = 'Tarball'
+
+name = 'Trim_Galore'
+version = '0.6.2'
+
+homepage = 'https://www.bioinformatics.babraham.ac.uk/projects/trim_galore/'
+description = """Trim Galore is a wrapper around Cutadapt and FastQC to
+consistently apply adapter and quality trimming to FastQ files, with extra
+functionality for RRBS data."""
+
+toolchain = {'name': 'GCCcore', 'version': '8.2.0'}
+
+source_urls = ['https://github.com/FelixKrueger/TrimGalore/archive/']
+sources = ['%(version)s.tar.gz']
+
+checksums = ['c50b841bdc294a6cdc6a27fb7bfbed1973541d20a68a4708584b817c58b3f376']
+
+multi_deps = {'Python': ['3.7.2', '2.7.15']}
+
+builddependencies = [('binutils', '2.31.1')]
+
+dependencies = [
+    ('Perl', '5.28.1'),
+    ('FastQC', '0.11.8', '-Java-11', True),
+    ('cutadapt', '1.18')
```

Comment:
@Darkless012 `pigz` dependency is missing? See existing easyconfig `Trim_Galore-0.6.2-foss-2018b-Python-3.6.6.eb`

`pigz` is only useful in combination with Python 3 (see documentation of the `-j` option at https://github.com/FelixKrueger/TrimGalore/blob/master/Docs/Trim_Galore_User_Guide.md), which is why it isn't included in the existing `Trim_Galore` easyconfig using Python 2.7.15...